### ISSUE TITLE
Change Ceph S3's Bucket Quota silently

### ISSUE THREAD

**Issue by IvanJobs** [intel-cloud/cosbench#311]
Hi, I'm using cosbench for ceph-s3 testing, after running complete, the job failed for Quota Exceed. So I modify ceph's quota to infinte, but running this job again, bucket quota is changed again, I'm wondering if cosbench modify bucket quota during running time?

**Comment by ywang19** [comment]
No. COSBench doesn’t change quota explicitly, but if quota is set by default when creating buckets, you will see quota change.
